tests/unit/test_configuration.py

```diff
@@ -107,6 +113,15 @@ def test_no_such_key_error_message_missing_option(self) -> None:
         with pytest.raises(ConfigurationError, match=pat):
             self.configuration.get_value("global.index-url")
 
+    def test_overrides_normalization(self) -> None:
```

Comment:
Note: This test has nothing to do with the change. I would happily remove it if it is a problem, but noticed that there was no unit test that combined precedent with normalization (both are well tested separately)